mvpa2/support/afni/lib_prep_afni_surf.py

```diff
@@ -286,19 +311,75 @@ def yesno2bool(d, k): # dict, key
             c['outvol_ext'] = '+' + c['outvol_view']
             c['outvol_fullext'] = c['outvol_ext'] + '.HEAD'
 
-
     return c
 
+
+
+def _ensure_expvol_is_plump(expvol):
+    '''Helper function that raises an error if the experimental volume
+        is oblique, because such volumes are not supported (yet)'''
+    cmd = '3dinfo "%s"' % expvol
```

Comment:
although it would require afni(3dinfo) so corresponding check for its presence then ideally could just go to externals.py